demo/components/presentational/EstateForSale.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { Image, StyleSheet, Text } from "react-native";
+import PropTypes from "prop-types";
+import {
+  responsiveWidth,
+  responsiveHeight,
+} from "react-native-responsive-dimensions";
+
+export default function EstateForSale({ estate }) {
```

Comment:
What about it matters that it's for sale here? Maybe just `Estate`?

Reply:
Yeah, I agree.

Reply:
Done.

Reply:
Thanks! Resolving.